### DISCUSSION TITLE
combine multiple smaller fixes in one pr or create separate ones for each change?

### DISCUSSION THREAD

**Discussion by b8daniel** [inovex/scrumlr.io#3382] [Q&A] [answered]
Hey, I'd like to work on 6 smaller css issues/edge cases I found - mostly small missalignments.
My question is, whether I should either... 

1. combine my proposed changes as one issue & pull request 
2. create multiple issues & pull requests
3. create multiple issues for the different findings but combine the fixes as one pr

personally, I'd take the 3rd, but I dont have much experience contributing to open source, so I wanted to ask here

**Comment by Resaki1** [comment, marked as answer]
I'd prefer option 3 as well, as I feel like this is how we have worked on this project in the past. Imo, creating multiple PRs for each small bug fix creates unnecessary overhead for everyone involved.
